src/util/hass-attributes-util.html

```diff
@@ -5,7 +5,7 @@
   binary_sensor: [
     'connectivity', 'light', 'moisture', 'motion', 'occupancy', 'opening',
     'sound', 'vibration', 'gas', 'power', 'safety', 'smoke', 'cold', 'heat',
-    'moving', 'plug', 'presence'],
+    'moving', 'plug', 'presence', 'battery'],
```

Comment:
Please sort

Reply:
Done, I also went back and sorted src/translations/en.json